migrations/20230726195558_add_mail_hash_field/migration.sql

```diff
@@ -0,0 +1,11 @@
+/*
+  Warnings:
+
+  - Added the required column `mailHash` to the `unregistered_notification_consent` table without a default value. This is not possible if the table is not empty.
+
+*/
+-- AlterTable
+ALTER TABLE "people" ADD COLUMN     "mailHash" TEXT;
```

Comment:
please see my comment for the generateHash function - it should be possible to not need this field in the database

Reply:
Removed